contrib/python-bindings/tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+## ---------------------------------------------------------------------
```

Comment:
My preference would be to put the python tests into `tests/python-bindings` to make them first-class citizens along with the other tests.